hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -171,7 +171,8 @@ public MerkleNetworkContext copy() {
 
 	@Override
 	public void deserialize(SerializableDataInputStream in, int version) throws IOException {
-		consensusTimeOfLastHandledTxn = serdes.readNullableInstant(in).toJava();
+		final var lastHandleTime = serdes.readNullableInstant(in);
+		consensusTimeOfLastHandledTxn = (lastHandleTime == null) ? null : lastHandleTime.toJava();
```

Comment:
Could be extracted to a helper function to remove code duplication when reuse and increase readability